docs/running-on-mesos.md

```diff
@@ -387,6 +387,13 @@ See the [configuration page](configuration.html) for information on Spark config
     </ul>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.coarse.shutdown.ms</code></td>
+  <td><code>10000</code> (10 seconds)</td>
+  <td> 
+    Time (in ms) to wait for executors to report that they have exited. Setting this too low has the risk of shutting down the Mesos driver (and thereby killing the spark executors) while the executor is still in the process of exiting cleanly.
```

Comment:
actually, I would probably not document this config. I don't really see a use case where users would want to change this. If we decide that users do want to control this (which I don't think they will) then we can always expose it later, but we can't go the other way because of backward compatibility.